modules/datastore/tests/src/Unit/Service/PostImportResultTest.php

```diff
@@ -42,33 +50,55 @@ public function testStoreJobStatus() {
       ->addMethods(['fields', 'execute'])
       ->getMock();
 
-    $queryMock->expects($this->once())
+    $queryMock->expects($this->any())
+      ->method('fields')
+      ->with([
+        'resource_identifier' => $dataResourceMock->getIdentifier(),
+        'resource_version' => $dataResourceMock->getVersion(),
+        'post_import_status' => 'done',
+        'post_import_error' => 'N/A',
+        'timestamp' => 1700000000,
+      ])
+      ->willReturnSelf();
+
+    $queryMock->expects($this->any())
+      ->method('execute')
+      ->willReturn(TRUE);
```

Comment:
Let's discuss this - I am really having trouble figuring out what is being tested here